.github/workflows/build_and_test_all.yml

```
@@ -60,8 +60,42 @@ jobs:
       if: matrix.os == 'windows-latest'
       run: cargo test
 
-    - uses: codecov/codecov-action@v2
+    - name: Upload Rust coverage
       if: matrix.os == 'macos-latest'
+      uses: codecov/codecov-action@v2
```

actionlint (1.7.12)

[error] the runner of "codecov/codecov-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)